src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestFactory.cs

```diff
@@ -349,7 +367,37 @@ private void AddHackerUser(List<ProCoSysProject> commonProCoSysProjects)
                         new TestProfile
                         {
                             FullName = "Sigurd Signer",
-                            Oid = _signerOid
+                            Oid = SignerOid
+                        },
+                    ProCoSysPlants = commonProCoSysPlants,
+                    ProCoSysPermissions = new List<string>
+                    {
+                        Permissions.COMMPKG_READ,
+                        Permissions.MCPKG_READ,
+                        Permissions.PROJECT_READ,
+                        Permissions.LIBRARY_FUNCTIONAL_ROLE_READ,
+                        Permissions.USER_READ,
+                        Permissions.IPO_READ,
+                        Permissions.IPO_SIGN,
+                        Permissions.IPO_WRITE,
+                        Permissions.IPO_ATTACHFILE,
```

Comment:
remove Permissions.IPO_ATTACHFILE and Permissions.IPO_DETACHFILE from all users accept Planner. Use Planner only for testing Upload and delete attachment